client/my-sites/add-ons/hooks/use-add-on-display-cost.ts

```diff
@@ -12,16 +12,16 @@ const useAddOnDisplayCost = ( productSlug: string, quantity?: number ) => {
 		const product = getProductBySlug( state, productSlug );
 
 		if ( product?.product_term === 'month' ) {
+			/* Translators: $formattedCost: monthly price formatted with currency */
 			return translate( '%(formattedCost)s/month, billed monthly', {
-				/* Translators: $formattedCost: monthly price formatted with currency */
 				args: {
 					formattedCost,
 				},
 			} );
 		}
 
+		/* Translators: $montlyCost: monthly price formatted with currency */
```

Comment:
The same goes here :) 
```suggestion
		/* Translators: %(montlyCost)s: monthly price formatted with currency */
```